### ISSUE TITLE
Can Header and footer be static in all pages

### ISSUE THREAD

**Issue by bangararaju** [eKoopmans/html2pdf#23]
In every page can we make static  Header and footer

**Comment by eKoopmans** [comment]
Hi @bangararaju, see #21. I'd recommend using a similar approach, taking the final jsPDF object and then adding your header and footer yourself. I'll look into adding something for headers/footers as a feature in the future, but it won't be soon.

**Comment by chrishaensel** [comment]
I use this in my callback function: 

```
function pdfCallback(pdfObject) {
    var number_of_pages = pdfObject.internal.getNumberOfPages()
    var pdf_pages = pdfObject.internal.pages
    var myFooter = "Footer info"
    for (var i = 1; i < pdf_pages.length; i++) {
        // We are telling our pdfObject that we are now working on this page
        pdfObject.setPage(i)
        // The 10,200 value is only for A4 landscape. You need to define your own for other page sizes
        pdfObject.text(myFooter, 10, 200)
    }
}
```

https://www.chaensel.de/jspdf-html2pdf-loop-through-pages-and-add-content/

**Comment by eKoopmans** [comment]
Hi, I have just pushed a new branch, [new-api](https://github.com/eKoopmans/html2pdf/tree/new-api), which is ready for testing! [The readme](https://github.com/eKoopmans/html2pdf/tree/new-api#advanced-usage) has all the information on the new API. Please let me know if there are any problems with it - I want to make sure it's stable before merging into master!

For this specific request, you could make changes to the PDF object before saving like so:

```js
html2pdf().from(element).toPdf().get('pdf').then(function (pdf) {
  // Your code to alter the pdf object.
}).save();
```

**Comment by eKoopmans** [comment]
Closing as resolved by the v0.9.0 new API. For anyone looking here, see [this post](https://github.com/eKoopmans/html2pdf/issues/21#issuecomment-447028134) for an example of adding headers/footers.
